plugins/mediafoundation/encoders/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.28...3.30)
+
+add_library(win-mf MODULE)
+add_library(OBS::win-mf ALIAS win-mf)
```

Comment:
Just a small heads-up, this name will need to change (here and throughout the plugin code) as it should distinguish itself from the Media Foundation-based capture plugin. I asked for guidance what that name should be, so that'd hopefully just be a final "refactor->rename" operation.